appkt/src/main/java/com/richpathanimator/sample/AnimationSamplesActivity.kt

```diff
@@ -95,10 +95,10 @@ class AnimationSamplesActivity : AppCompatActivity() {
 
     private fun animateArrowToSearch() {
 
-        val searchCircle = icArrowSearchRichPathView.findRichPathByName("search_circle")
-        val stem = icArrowSearchRichPathView.findRichPathByName("stem")
-        val arrowTop = icArrowSearchRichPathView.findRichPathByName("arrow_head_top")
-        val arrowBottom = icArrowSearchRichPathView.findRichPathByName("arrow_head_bottom")
+        val searchCircle = icArrowSearchRichPathView.findRichPathByName("search_circle")!!
+        val stem = icArrowSearchRichPathView.findRichPathByName("stem")!!
+        val arrowTop = icArrowSearchRichPathView.findRichPathByName("arrow_head_top")!!
+        val arrowBottom = icArrowSearchRichPathView.findRichPathByName("arrow_head_bottom")!!
```

Comment:
usually, we shouldn't accept `!!` here but let's say it's OK because we could consider this sample app as a test app

Reply:
@tarek360 I think so. I'll modify this on tonight.

Reply:
@tarek360 
I think if each Richpath is null, then we should prevent going to next step like this:
`val searchCircle = icArrowSearchRichPathView.findRichPathByName("search_circle") ?: return`

What do you think of this?